summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormTopologyBuilder.scala

```diff
@@ -12,6 +13,25 @@ import org.apache.storm.generated.StormTopology
 import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
 
+import scala.collection.{ Map => CMap }
+
+object StormTopologyBuilder {
+  type Item[T] = (Timestamp, T)
+
+  type AggregatedKey[K] = (K, BatchID)
+  type AggregatedValue[V] = (Timestamp, V)
+  type PartiallyAggregated[K, V] = (Int, CMap[AggregatedKey[K], AggregatedValue[V]])
```

Comment:
This case is kinda different. For key and value we want to highlight this is a type we want to aggregate over. For `PartiallyAggregated` this is a type of aggregated result, so I prefer to keep it to be `PartiallyAggregated`